config/global_config/sandbox.sh

```diff
@@ -1,4 +1,5 @@
 CONFIG=sandbox
+ENVIRONMENT=sandbox
```

Comment:
Do we need to add one for [review.sh](https://github.com/DFE-Digital/register-early-career-teachers-public/blob/main/config/global_config/review.sh)?

Reply:
I don't think so, `ENVIRONMENT` is set per pull request (i.e., `REVIEW-123`) [in the Makefile](https://github.com/DFE-Digital/register-early-career-teachers-public/blob/main/Makefile#L26).